.eslintrc.js

```diff
@@ -3,34 +3,22 @@ module.exports = {
   extends: ['@react-native-community', 'prettier'],
```

Comment:
I think you have prettier or eslint installed globally and it may mess the new lines, quotes, etc.

In this file only the change for additional hooks should be present in the diff

Reply:
Yup, all good, thanks!